benchexec/tools/coveriteam-verifier-validator.py

```diff
@@ -29,8 +29,8 @@ def cmdline(self, executable, options, task, rlimits):
         data_model_param = get_data_model_from_task(
             task, {ILP32: "ILP32", LP64: "LP64"}
         )
-        if data_model_param and "--data-model" not in options:
-            options += ["--data-model", data_model_param]
+        if data_model_param and "data_model" not in options:
```

Comment:
Wild guess: Isn't this wrong?

I expect 'options' to be a list similar to
`["myComposition.cvt", "--input", "data_model=ILP32"]`

So this would never contain "data_model".

Maybe you need something like this?
`if data_model_param and not any(opt.startswith("data_model=") for opt in options)`

Reply:
Looks good 👍 Did you test it?

Reply:
What exactly is actually the format that CoVeriTeam supports for these options? For example, if CoVeriteam supports `--input "data_model = foo"` (like CPAchecker does), then the new check would also not always match correctly.

Reply:
> Leading whitespace before like `ILP32` leads to a crash.

Sounds like a bug to fix in CoVeriTeam ;-)

>  I'm not sure now, how to check for an existing data_model. We can also check for `data_model =`, but I think we need the `=`, otherwise `data_model_program_one` would block the adding of the input too

I agree. But quite likely CoVeriTeam accepts any number of spaces before the `=`. So it might need a regex like `^data_model\s*=` or `^data_model *=`.

Reply:
This is not a question for BenchExec or the tool-info module, and it is not about data models in particular. If CoVeriTeam handles `foo = bar` the same as `foo=bar`, when we need to do so as well.

CoVeriTeam needs to decide whether using `foo= bar` sets the value of `foo` to `bar` or to ` bar`.